pkg/execution/computionadmission/admission.go

```diff
@@ -219,7 +219,7 @@ func (ac *AdmissionContext) checkPRGBRequired(ar *admission_v1beta1.AdmissionReq
 
 	data := cfgMap.Data[orch_meta.ConfigMapConfigKey]
 	var config orch_meta.ServiceProperties
-	err = yaml.Unmarshal([]byte(data), &config)
+	err = yaml.UnmarshalStrict([]byte(data), &config)
```

Comment:
this maybe unnecessary? what if we add/remove some field in `service-metadata`? this may break a webhook (depending on deployment order)

Reply:
You are right, I haven't verified each usage place. I'll do that.